dactyl-config.yml

```diff
@@ -4100,6 +4100,13 @@ pages:
         targets:
             - en
 
+    -   name: XRPL Jobs
+        html: http://jobs.xrpl.org/
```

Comment:
Let's upgrade this to HTTPS as a good practice.

```suggestion
        html: https://jobs.xrpl.org/
```

Reply:
Agreed. I should have added that